drivers/chown_unix.go

```diff
@@ -45,10 +45,20 @@ func platformLChown(path string, info os.FileInfo, toHost, toContainer *idtools.
 			uid, gid = mappedPair.UID, mappedPair.GID
 		}
 		if uid != int(st.Uid) || gid != int(st.Gid) {
+			stat, err := os.Lstat(path)
```

Comment:
Maybe I'm missing something here, but I don't understand the reason for calling `os.Lstat()` here -- this function is already called with the `info` parameter, which is result of `os.Lstat(path)`.

Reply:
good catch, I agree we can avoid `Lstat` here